proto/cacheclient.proto

```diff
@@ -70,7 +70,7 @@ message _DictionaryGetResponse {
   string message = 2;
 }
 
-message _DictionaryGetAllRequest {
+message _DictionaryFetchRequest {
```

Comment:
We also need to update the `DictionaryGetResponse` and `DictionaryFetchResponse` to the `Found/Missing` format like we did for `SetFetch` and also remove the `ECacheResult` from all the dictionary api responses.

Reply:
Added `Found/Missing` in caf8c1a.